lib/middleware.js

```diff
@@ -79,7 +79,7 @@ module.exports = function wrapper(context) {
           contentType += '; charset=UTF-8';
         }
 
-        if (!res.getHeader('Content-Type')) {
+        if (!res.getHeader || (typeof res.getHeader === 'function' && !res.getHeader('Content-Type'))) {
```

Comment:
Can we simplify this? Looks very misleading, like:
```
if (res.getHeader && !res.getHeader('Content-Type')) {
}
```

Reply:
I think the question to be answered first is: What to do in the case where `res.getHeader` is not available. Execute the `if` statement or not?

@sjy in your statement you will execute the `if` statement when `res.getHeader` is not available. Is it correct, that **set** Header function is available though?

Reply:
since the use case only effects  project using the latest webpack-koa which will be fixed, this pr is will be closed.  @dmohns, thanks your attention, code changes here is more like a hot-fix.

Reply:
since the use case only effects  project using the latest webpack-koa which will be fixed, this pr is will be closed.  @dmohns, thanks your attention, code changes here is more like a hot-fix.